### ISSUE TITLE
EtherDelta Not Recognizing Metamask After Password Change

### ISSUE THREAD

**Issue by dcdalton** [MetaMask/metamask-extension#6641]
I hadn't logged into EtherDelta in some time and I recently 
change my MM password. Today I trying to sync with ED and no matter what I do I get the message "You are using MetaMask but you are not logged in. Please log in to MetaMask and refresh." ... I have read every thread here (and other places) I could find and tried all the suggestions but for the life of me I can't fix this the networks I have available in MM are the main Ethereum network,  the test networks, localhost 8545 and 'custom rpc'

How the heck to I get this thing to sync with ED?

**Comment by tmashuang** [comment]
https://github.com/MetaMask/metamask-extension/issues/6579

TLDR; Settings > Security & Privacy > disable Privacy Mode.

**Comment by Aldekein** [comment]
This didn't help me and I had to switch to https://forkdelta.app/ to get access to my 0.5 ETH stuck in orders 🤑
